controllers/cluster_controller.go

```diff
@@ -699,6 +699,32 @@ func (r *FoundationDBClusterReconciler) hasFullReplication(cluster *fdbtypes.Fou
 	return status.Cluster.FullReplication, nil
 }
 
+func (r *FoundationDBClusterReconciler) hasDesiredFaultTolerance(cluster *fdbtypes.FoundationDBCluster) (bool, error) {
+	adminClient, err := r.DatabaseClientProvider.GetAdminClient(cluster, r)
```

Comment:
If the version of FDB is lower than 6.2 just approve it since we are missing the zone fault tolerance information and 6.1 will be deprecated in 1.0.